### ISSUE TITLE
Python 2 Compatibility

### ISSUE THREAD

**Issue by aditya95sriram** [mikepound/pwned-search#7]
Since this tool might be used by people other than programmers, complete compatibility with Python2 is a desirable feature. Almost all the code is already backwards compatible and the only (minor) issue is in the two `print` statements in the `main` function ([line 39](https://github.com/mikepound/pwned-search/blob/521c441a6add98892a6910645894a4aa8d60bcc3/pwned.py#L39) and [line 43](https://github.com/mikepound/pwned-search/blob/521c441a6add98892a6910645894a4aa8d60bcc3/pwned.py#L43)), which, in Python2, currently print something like 

    ('Password1', 'was found') 

 These can be easily modified to use string formatting so as to make the code 100% compatible with Python2 and produce the same results regardless of the version.

In other words, change `print(pwd, "was found")` to `print("{} was found".format(pwd))` and `print(pwd, "was not found")` to `print("{} was not found".format(pwd))`

**Comment by ToonSpin** [comment]
If those are the only two changes, why not make a quick PR?

**Comment by aditya95sriram** [comment]
@ToonSpin Just wanted to check if it's worth it, this can be deemed borderline pedantic. Also I didn't think about versions of Python older than Python2.6, so putting this out here increases the chances of finding other bugs (if they exist) in the older versions, and then all the fixes can be incorporated in one go.

**Comment by mikepound** [comment]
Hi, thanks for this. Perfectly happy for you to tweak the code in a PR and i'll merge it. Please make sure it remains compatible with Python 3! :)

**Comment by micsthepick** [comment]
#11 implements these changes, while maintaining compatibility with Python 3.

**Comment by mikepound** [comment]
Thanks! 👍

**Comment by aditya95sriram** [comment]
Sorry to bring it up again, but I think #12 kind of breaks this with

    #!/usr/bin/env python3

As I have both Python 2 and Python 3 installed on my system so I don't know how that shebang behaves on a system with just Python 2 installed. Although, as per this [StackOverflow answer](https://stackoverflow.com/a/19305076/1614140) I believe it's safer to use the following shebang (we qualify for the _rare case_ mentioned there)

    #!/usr/bin/env python

**Comment by ToonSpin** [comment]
As long as the script is both Python 2 and 3 compatible I do believe that @aditya95sriram's is a good shebang line to use.

**Comment by aditya95sriram** [comment]
@ToonSpin And keeping the script Python 2 compatible is what I'm proposing. Because the target users of this repo need not be programmers and might be people who stick with the default Python installation their system came with, which in many cases is Python 2.

**Comment by ToonSpin** [comment]
I was wholeheartedly agreeing with you, that's all. :) My opinion probably doesn't really matter because it's not my repo, but you were saying sorry for bringing it up, and I felt that you were right, so I wanted to show support.

My comment is trying to say that it's not *always* the right shebang, but because you and Mike feel things should be Python 2 and 3 compatible (which I think is a good thing), I think changing the shebang is the right thing to do, but I guess it didn't come out right.

**Comment by aditya95sriram** [comment]
@ToonSpin no offence taken 😄

**Comment by mikepound** [comment]
You're both right - I didn't spot the 3! I'll make the change, it's best to preserve compatibility, most installations will have a symbolic link between python to whichever anyway.

**Comment by aditya95sriram** [comment]
As the SO answer states
> The reason for these recommendations, given in PEP 394, is that python can refer either to python2 or python3 on different systems.

Whichever it is, it can only refer to an installed version and since we're covering all our bases this shebang is perfect for us.

Also to prevent issues like this from happening in the future, maybe you should add a line in the readme asking people to preserve compatibility in their PRs.
